stock_pack_operation_quick_lot/views/stock_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2018 Carlos Dauden - Tecnativa <carlos.dauden@tecnativa.com>
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+
+<odoo>
+
+<record id="view_picking_form" model="ir.ui.view">
+    <field name="name">stock.picking.form</field>
+    <field name="model">stock.picking</field>
+    <field name="inherit_id" ref="stock.view_picking_form"/>
+    <field name="arch" type="xml">
+        <xpath expr="//field[@name='pack_operation_product_ids']/tree//field[@name='owner_id']"
+               position="after">
+            <field name="lot_track_enabled" invisible="1"/>
+            <field name="lot_name"
```

Comment:
Shouldn't this be visible only on incoming pickings?

Reply:
Why?

Reply:
Because you are not creating a lot on outgoing. Lot name can be on outgoing, but definitively not end date.

Reply:
I try hide it